tests/createContactlessTask.test.ts

```diff
@@ -101,7 +111,8 @@ describe('createContactlessTask', () => {
     };
 
     const { getTwilioClient, DOMAIN_NAME } = baseContext;
-    await createContactlessTask({ getTwilioClient, DOMAIN_NAME }, event1, callback1);
+    const payload: any = { getTwilioClient, DOMAIN_NAME };
+    await createContactlessTask(payload, event1, callback1);
```

Comment:
Was this refactor needed? Logically seems like doing the same to me.

Reply:
@GPaoloni  Yes, the refactor was needed. createContactlessTask was expecting three parameters and not an object as part of the parameter. So Typescript wasn't allowing it to fly. I had to refactor the object to a single variable.